### PR TITLE
Remove the values app

### PR DESCRIPTION
**Pull request by rowanseymour**
This now just a set of constants which belong in `ContactField`

### CODE REVIEW

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -56,7 +55,7 @@ def normalize_key(key):
         return INVALID_EXTRA_KEY_CHARS.sub("_", key)[:255]
 
     if isinstance(extra, str):
-        return extra[: Value.MAX_VALUE_LEN], count + 1
+        return extra[:640], count + 1
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py:58]: this whole method is a bit weird and I wonder if we can't get rid of it and just say that there's a limit on the *size* of extra.. in terms of characters

File: temba/api/v2/serializers.py [hunk 2]
```diff
@@ -659,20 +658,27 @@ def save(self):
...
+        ContactField.TYPE_STATE: "state",
+        ContactField.TYPE_DISTRICT: "district",
+        ContactField.TYPE_WARD: "ward",
+    }
 
     value_type = serializers.SerializerMethodField()
 
     def get_value_type(self, obj):
-        return self.VALUE_TYPES.get(obj.value_type)
+        return self.VALUE_TYPES[obj.value_type]
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py:673]: better to blow up with a sentry error than not know we have weird field types out there

File: temba/api/v2/serializers.py (hunk 1 above)
Comment (nicpottier) [reply temba/api/v2/serializers.py:58]: I think the argument here is for Zapier. In that users are "using" the API there without a lot of technical knowledge and having a call fail entirely when they have some input that is larger than our extra could be confusing.

Maybe we just up this limit some? Extra can be pretty gigantic these days can't it?

**Review by nicpottier — APPROVED** [review]
Finally Values are gone too!

File: temba/api/v2/serializers.py (hunk 1 above)
Comment (rowanseymour) [reply temba/api/v2/serializers.py:58]: This specific line is the limit on the character length of string in the JSON.. which definitely feels weird (in the non-Zapier case) to just truncate silently. The only technical limitation on params/extra size is that it ends up in the session blob being serialized/de-serialized - the engine will accept whatever you give it.

Comment (rowanseymour) [reply temba/api/v2/serializers.py:58]: Could say that Zapier passes a param to do this automatic truncating (easy mode).. everyone else gets a character limit and error messages (advanced mode)

**Pull request merged**